src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -95,6 +94,11 @@ trait Backend {
    */
   def stdoutStderr(descriptor: WorkflowDescriptor, callName: String, index: ExecutionIndex): CallLogs
 
+  /**
+   * Provides a function that given a WdlFile, returns its hash.
```

Comment:
Yes I can see why it's confusing, that's why I though maybe "Returns" instead of "Provides" would be a bit clearer ?